packages/kbn-reporting/server/export_type.ts

```diff
@@ -55,7 +55,7 @@ export abstract class ExportType<
   abstract jobContentEncoding?: 'base64' | 'csv';
   abstract jobContentExtension: 'pdf' | 'png' | 'csv';
 
-  abstract createJob: CreateJobFn<JobParamsType>;
+  abstract createJob: CreateJobFn<JobParamsType, TaskPayloadType>;
```

Comment:
The `CreateJobFn` type is meant to accept 2 generic arguments. This is a correction to make sure that all `createJob` functions for export types in the reporting system unanimously do:
1. Accept job params from the user request
2. Return a correctly typed payload for storing in the reporting system index / queuing with Task Manager.